src/main/java/seedu/address/model/book/Book.java

```diff
@@ -17,9 +17,12 @@
  */
 public class Book {
 
+    public static final String TIME_ADDED_MESSAGE_CONSTRAINTS = "Time added should be able to be parsed to long.\n";
+
     // Identity fields
     private final BookName bookName;
     private final Isbn isbn;
+    private final long timeAdded;
```

Comment:
Good job on introducing the timeAdded attribute. It will aid us in differentiating the books with the same bookName, isbn, authors and tags. I will accommodate accordingly on my end for the changes.